portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,4 +29,4 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello Matt!</h1>");
```

Comment:
Minor: Since we are now just returning some data value (for the content), not a whole HTML, these two lines could have been changed (different `setContentType()` parameter, which I'm afraid I don't know for sure, and different `println()` value without the HTML tag `<h1>`), but I'm pretty sure you'll be doing that in the next steps, so no need to do that now.